src/VRCFaceTracking.Avalonia/Controls/SettingsExpanderToggle.axaml

```diff
@@ -17,12 +18,19 @@
         <Expander.Header>
             <Grid>
                 <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="Auto" />
                     <ColumnDefinition Width="Auto" />
                     <ColumnDefinition Width="*" />
                     <ColumnDefinition Width="Auto" />
                 </Grid.ColumnDefinitions>
+                <PathIcon Data="{Binding IconPath, RelativeSource={RelativeSource AncestorType=controls:SettingsExpanderToggle}, Converter={x:Static converters:TypeConverters.IconConverter}}"
+                          Grid.Column="0"
+                          Background="#00000000"
```

Comment:
You should be able to remove a fully transparent background from a PathIcon, and it won't render.